kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -75,22 +74,36 @@
         v-model="confirmedPassword"
       />
 
-      <ui-select
-        :name="$tr('selectFacility')"
-        :placeholder="$tr('selectFacility')"
-        :label="$tr('facility')"
-        :value="selectedFacility"
-        :options="facilityList"
-        :invalid="facilityIsInvalid"
-        :error="facilityIsInvalidText"
-        @blur="facilityBlurred = true"
-        @input="updateSelection"
+      <!-- ToS modal only shows up if the box empty prior to clicking -->
+      <!-- Using UI Checkbox for blur events and v-model -->
+      <k-checkbox
+        id="terms-agreement-checkbox"
+        :class="['terms-agreement-checkbox', termsNotAgreed ? 'invalid' : '']"
+        :checked="termsAgreed"
+        @change="showTerms = termsAgreed = $event"
+        @blur="termsAgreementCheckboxBlurred = true"
```

Comment:
The k-checkbox does not emit a blur event, so we will have to add that to the 0.6.1 branch

Reply:
Forgot to add that change, but have it locally. I'll submit a PR in a bit.